low-level/files/opt/seagate/sspl/setup/sspl_cleanup.py

```diff
@@ -28,8 +28,8 @@
 from framework.base.sspl_constants import (
     SSPL_CONFIG_INDEX, file_store_config_path, global_config_file_path,
     sspl_config_path, PRODUCT_BASE_DIR, SSPL_BASE_DIR, PRODUCT_FAMILY,
-    RSYSLOG_IEM_CONF, RSYSLOG_SSPL_CONF, RSYSLOG_MSB_CONF,
```

Comment:
so Cortx logger takes care of log rotation already ! Where is its config, where component specific preferences could be set !?

Reply:
> so Cortx logger takes care of log rotation already
yes

> Where is its config, where component specific preferences could be set !?
These are the default config params,
backup_count=10,
file_size_in_mb=10,

We can parse these params in init method, to change the config values